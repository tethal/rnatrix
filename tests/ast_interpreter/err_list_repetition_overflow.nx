fun main() {
    // This should trigger overflow check: trying to allocate huge list
    var large = [1, 2] * 10;
    print(large * 999999999999999999);
}