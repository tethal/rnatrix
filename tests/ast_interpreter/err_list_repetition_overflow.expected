<string>:4:17: error: list repetition result too large
    print(large * 999999999999999999);
                ^
